components/hal/emac_hal.c

```diff
@@ -517,6 +517,10 @@ esp_err_t emac_hal_ptp_set_target_time(emac_hal_context_t *hal, uint32_t seconds
     emac_ll_ts_target_int_trig_enable(hal->ptp_regs);
     return ESP_OK;
 }
+esp_err_t emac_hal_ptp_enable_ts4all(emac_hal_context_t *hal, bool enable) {
```

Comment:
Fix formatting, there should be space before function header. Did you run pre-commit hook? If not, please check https://docs.espressif.com/projects/esp-idf/en/stable/esp32/contribute/install-pre-commit-hook.html. It should help you to catch and fix this type of errors.